pages/common/lb.md

```diff
@@ -0,0 +1,28 @@
+# lb
+
+> A blog management tool.
+> More information: <https://github.com/LukeSmithxyz/lb>.
+
+- Make a new draft:
+
+`./lb new`
+
+- Edit a draft:
```

Comment:
Which draft does that edit? Or can you only have one at a time?

Reply:
If you have multiple drafts, it has a kind of wizard, like this:
```
1 first-article
2 second-article

Select an article to edit by id.
```

As of [July 2020](https://github.com/LukeSmithxyz/lb/commit/f13cf1a3a642dcaa6eb188b15db4e36450edcabd), if you only have one draft, it automatically selects that one.